packages/core/src/rum/instrumentation/resourceTracking/implementation/__tests__/XHRProxy.test.ts

```diff
@@ -229,20 +244,25 @@ describe('XHRPr', () => {
             await flushPromises();
 
             // THEN
-            const traceId = xhr.requestHeaders[TRACE_ID_HEADER_KEY];
-            expect(traceId).toBeDefined();
-            expect(traceId).toMatch(/[1-9].+/);
+            expect(xhr.requestHeaders[TRACE_ID_HEADER_KEY]).toBeUndefined();
+            expect(xhr.requestHeaders[PARENT_ID_HEADER_KEY]).toBeUndefined();
         });
 
-        it('generates different ids for spanId and traceId in request headers', async () => {
+        it('does not generate spanId and traceId in request headers when no the url does not match first party hosts', async () => {
```

Comment:
probably a typo here

```suggestion
        it('does not generate spanId and traceId in request headers when the url does not match first party hosts', async () => {
```